### PR TITLE
Updated German translation to 1.3.0

### PR DESCRIPTION
**Pull request by morkro**
:de:

### CODE REVIEW

File: _includes/snippets/architecture/01/de.html [hunk 1]
```diff
@@ -3,50 +3,56 @@
...
 |
 |– themes/
-|   |– _theme.scss       # Default-Theme
-|   |– _admin.scss       # Admin-Theme
-|   …                    # Etc.
+|   |– _theme.scss        # Default-Theme
+|   |– _admin.scss        # Admin-Theme
+|   …                     # Etc.
 |
 |– utils/
```
Comment (HugoGiraudel) [inline _includes/snippets/architecture/01/de.html]: So this `utils` block should actually be removed.

File: de/_architecture.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ Zum Glück ist einer der Hauptvorteile eines CSS Präprozessor der, eine Codebas
...
 
-Es gibt bereits viele bekannte Frameworks für CSS Projekte: [OOCSS](http://oocss.org/), [Atomic Design](http://bradfrost.com/blog/post/atomic-web-design/) sowie ein [Bootstrap](http://getbootstrap.com/) und [Foundation](http://foundation.zurb.com/) ähnliches. Alle mit ihren Vor- und Nachteilen.
+Es gibt [viele beliebte Architekturen](http://www.sitepoint.com/look-different-sass-architectures/) für CSS Projekte: [OOCSS](http://www.smashingmagazine.com/2011/12/12/an-introduction-to-object-oriented-css-oocss/), [Atomic Design](http://bradfrost.com/blog/post/atomic-web-design/) sowie ein [Bootstrap](http://getbootstrap.com/) und [Foundation](http://foundation.zurb.com/) ähnliches... alle mit ihren Vor- und Nachteilen.
```
Comment (HugoGiraudel) [inline de/_architecture.md]: Maybe use `…` instead of `...`.

File: de/_architecture.md [hunk 3]
```diff
@@ -43,7 +58,7 @@ Zurück zur Architektur, oder? Ich arbeite normalerweise mit einem Muster was ic
 * `layout/`
 * `pages/`
 * `themes/`
-* `utils/`
+* `abstracts/`
```
Comment (HugoGiraudel) [inline de/_architecture.md]: This should be moved first item in the list.

File: de/_architecture.md [hunk 4]
```diff
@@ -73,7 +88,7 @@ Der `base/` Ordner beinhaltet etwas wie das Boilerplate des Projekts. Da wird ei
 * `_typography.scss`
 
 <div class="note">
-	<p>Falls dein Projekt <em>eine Menge</em> CSS Animationen verwendet, kannst du dir überlegen eine <code>_animations.scss</code> mit allen <code>@keyframes</code> Definitionen von deiner Animationen hinzuzufügen. Falls du sie aber nur sporadisch benutzt, würde ich sie bei den jeweiligen Selektoren behalten.</p>
+	<p>Falls dein Projekt <em>eine Menge</em> CSS Animationen verwendet, kannst du dir überlegen eine <code>\_animations.scss</code> mit allen <code>@keyframes</code> Definitionen von deiner Animationen hinzuzufügen. Falls du sie aber nur sporadisch benutzt, würde ich sie bei den jeweiligen Selektoren behalten.</p>
```
Comment (HugoGiraudel) [inline de/_architecture.md]: Indentation a bit too large maybe?

File: _includes/snippets/architecture/01/de.html (hunk 1 above)
Comment (morkro) [inline _includes/snippets/architecture/01/de.html]: Yeah sorry, I guess I got a little confused a some point with the changelog  :see_no_evil:

File: de/_architecture.md (hunk 4 above)
Comment (morkro) [inline de/_architecture.md]: Team tabs checking in! Changed all of them to spaces :v:

File: _includes/snippets/architecture/01/de.html (hunk 1 above)
Comment (HugoGiraudel) [inline _includes/snippets/architecture/01/de.html]: My fault.

**Comment by HugoGiraudel** [comment]
Alright, thank you Moritz! :)

**Pull request merged**